var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -242,13 +249,37 @@ def check_compilers(self):
         # require both.
         # As of 08/2022 (0.3.21), we can build purely with a C compiler using
         # a f2c translated LAPACK version
-        #   https://github.com/OpenMathLib/OpenBLAS/releases/tag/v0.3.21
+        #   https://github.com/xianyi/OpenBLAS/releases/tag/v0.3.21
         if self.compiler.fc is None and "~fortran" not in self.spec:
             raise InstallError(
                 self.compiler.cc
                 + " has no Fortran compiler added in spack. Add it or use openblas~fortran!"
             )
 
+    @property
+    def headers(self):
+        # The only public headers for cblas and lapacke in
+        # openblas are cblas.h and lapacke.h. The remaining headers are private
+        # headers either included in one of these two headers, or included in
+        # one of the source files implementing functions declared in these
+        # headers.
+        return find_headers(["cblas", "lapacke"], self.prefix.include)
+
+    @property
+    def libs(self):
+        spec = self.spec
+
+        # Look for openblas{symbol_suffix}
+        name = ["libopenblas", "openblas"]
+        search_shared = bool(spec.variants["shared"].value)
+        suffix = spec.variants["symbol_suffix"].value
+        if suffix != "none":
+            name += suffix
```

Comment:
How does this work? Concatenating a string and a list like this results in the string being decomposed into chars and each char being added to the list, so at best this is a no op polluting the lib list?
@scheibelp

Reply:
Sorry to comment on a dead PR, I just went to look at this for the PATH issue when considering how to handle virtual deps and this jumped out at me.
Happy to submit a patch if I I'm not wrong about this. I've got one locally.

Reply:
> How does this work?

It almost definitely doesn't. A patch would be great.

Reply:
Ok, one will be up shortly!